docs/Documentation/Systems/Vermillion/applications.md

```diff
@@ -5,4 +5,69 @@ parent: Vermillion
 grand_parent: Systems
 ---
 
+The Vermilion HPC cluster marries traditional HPC deployments and modern cloud architectures.
 # Applications
+A list of packages available from the OpenHPC Community repo can be found at:
+
+    http://repos.openhpc.community/OpenHPC/2/EL_8/aarch64/
+
+Vermilion currently includes a subset of these packages in the base deployment. These include:
```

Comment:
This list could very easily become out of date. Is there a better way to veiw/caputre available packages?